### PR TITLE
[WIP] Issue #5215: smdba updates

### PR DESCRIPTION
**Pull request by Loquacity**
From feedback:
* updated list of files
* made recovery into a procedure

### CODE REVIEW

File: adoc/bp_chap_suma_backup.adoc [hunk 1]
```diff
@@ -64,55 +64,59 @@ Running out of space can result in backups failing, so check this often.
...
-* [path]``/srv/www/htdocs/pub/``
-* [path]``/var/spacewalk/packages/``
-** Each item within this directory represents an organization using custom RPMs, ensure you back up each of them
+* [path]``/etc/sysconfig/rhn/``
 * [path]``/root/.gnupg/``
+* [path]``/root/.ssh``
+** This file exists if you are using an SSH tunnel or SSH [command]``push``. You will also need to have saved a copy of the ``id-susemanager`` key.
 * [path]``/root/ssl-build/``
-* [path]``/etc/dhcp.conf``
+* [path]``/srv/salt``
```
Comment (mcalmer) [inline adoc/bp_chap_suma_backup.adoc]: We also need /srv/pillar

File: adoc/bp_chap_suma_backup.adoc [hunk 2]
```diff
@@ -64,55 +64,59 @@ Running out of space can result in backups failing, so check this often.
...
-* [path]``/etc/cobbler/``
+* [path]``/srv/www/cobbler``
+* [path]``/srv/www/htdocs/pub/``
 * [path]``/var/lib/cobbler/``
 * [path]``/var/lib/rhn/kickstarts/``
-* [path]``/srv/www/cobbler``
-* [path]``/root/.ssh``
-** This file exists if you are using an SSH tunnel or SSH [command]``push``. You will also need to have saved a copy of the ``id-susemanager`` key.
+* [path]``/var/spacewalk/``
+* [path]``/var/spacewalk/packages/``
```
Comment (mcalmer) [inline adoc/bp_chap_suma_backup.adoc]: Do we need to name `packages` if we already mirror full /var/spacewalk ?

**Comment by mcalmer** [comment]
@Loquacity JFYI: we use `[WIP]` in the title of a PR to indicate it is not ready.

File: adoc/bp_chap_suma_backup.adoc (hunk 2 above)
Comment (Loquacity) [reply adoc/bp_chap_suma_backup.adoc]: That's a good point, probably not.

**Comment by Loquacity** [comment]
@mcalmer any more updates for me?

**Comment by mcalmer** [comment]
@jcayouette We suggest to backup /var/spacewalk. If we do this, we can ignore any directory inside of it as it will be backup-ed . And /etc/sysconfig/rhn is still in the list of things which should be in the backup.

@Loquacity From my point of view we are complete. If nobody else has ideas we can merge.

**Comment by Loquacity** [comment]
Thanks @mcalmer :)

**Pull request merged**